src/WebUI/WorkLoads/DeploymentsTable.tsx

```diff
@@ -59,10 +57,11 @@ export class DeploymentsTable extends BaseComponent<IDeploymentsTableProperties,
 
         this._store.addListener(WorkloadsEvents.ReplicaSetsFetchedEvent, this._onReplicaSetsFetched);
         this._imageDetailsStore.addListener(ImageDetailsEvents.HasImageDetailsEvent, this._setHasImageDetails);
+        KubeFactory.telemetryService.scenarioStart(Scenarios.Deployments);
```

Comment:
why this is moved to end of the constructor? every where else it is start of it.

Reply:
Changed to let the child mark parent's TTI when rendered. Removing starting the scenario in child component